src/core/Akka.TestKit/TestKitBase.cs

```diff
@@ -710,10 +718,41 @@ public IActorRef CreateTestActor(string name)
             return CreateTestActor(_testState.System, name);
         }
 
+        private IActorRef CreateInitialTestActor(ActorSystem system, string name)
```

Comment:
we have a separate method for creating our implicit / default `TestActor` that doesn't share a code path with the others